### PR TITLE
Add website

### PR DESCRIPTION
**Pull request by pseudo-rnd-thoughts**
The initial setup for a website using the same style as all other farama websites

Todo
- [ ] Autobuild the environment documentation
- [x] Read the new documentation to check it is consistent
- [x] Fix the environment images / gifs
- [ ] Auto generate the gifs
- [x] Add github action 

I'm not going to implement the first or fourth items as the rooms don't work currently within the CI I believe

### CODE REVIEW

**Comment by pseudo-rnd-thoughts** [comment]
@JesseFarebro Could you have a quick look, you should be able to preview the website by installing the requirements from `docs/requirements.txt` then running `make html` in docs

File: .github/workflows/build-docs.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: Build main branch documentation website
+
+on:
+  push:
+    branches: [main]
```
Comment (mgoulao) [inline .github/workflows/build-docs.yml:5]: This repo uses `master` instead of `main`

Comment (pseudo-rnd-thoughts) [reply .github/workflows/build-docs.yml:5]: Thanks

**Review by jjshoots — APPROVED** [review]
LGTM.

**Pull request merged**